### PR TITLE
Fixes issues with the System.Net.Http tests

### PR DESCRIPTION
**Pull request by mmitche**
* Tests were not running because the directories were not named properly.
* Rename test project to match the correct name.
* Fix typo in resource strings.

### CODE REVIEW

File: src/System.Net.Http/src/Resources/Strings.resx [hunk 1]
```diff
@@ -198,8 +198,8 @@
   <data name="net_http_client_send_error" xml:space="preserve">
     <value>An error occurred while sending {0}. {1}</value>
   </data>
-  <data name="net_http_client_send_canceled" xml:space="preserve">
-    <value>Request for {0} was canceled.</value>
+  <data name="net_http_client_send_cancelled" xml:space="preserve">
+    <value>Request for {0} was cancelled.</value>
```
Comment (davidsh) [inline src/System.Net.Http/src/Resources/Strings.resx]: This is not a misprint.  It was explicitly spelled that way (with one "L").  There are two accepted spellings of the word "canceled".  It turns out that much of the .NET Framework is inconsistent on this.

**Comment by davidsh** [comment]
cc: @weshaggard , @ericstj @SidharthNabar

File: src/System.Net.Http/src/Resources/Strings.resx (hunk 1 above)
Comment (mmitche) [inline src/System.Net.Http/src/Resources/Strings.resx]: Okay will revert this.

Comment (stephentoub) [inline src/System.Net.Http/src/Resources/Strings.resx]: Yes.  We've tried to standardized on one 'l' (except in "cancellation", which has two 'l's), e.g. "OperationCanceledException".  There was some usage in the framework that predated that, though.

**Comment by stephentoub** [comment]
@dotnet-bot test this please

**Comment by mmitche** [comment]
@stephentoub Was this to retest after project.lock.json fixes?  If so, I think I have to merge to head and push again.

**Comment by stephentoub** [comment]
It was to re-test after I checked in #1974.

**Comment by mmitche** [comment]
@stephentoub yeah, this will just retest the old merge. Let me remerge first.

**Comment by stephentoub** [comment]
Looks like 10 tests are failing on Linux, mostly related to globalization.

**Comment by davidsh** [comment]
@mmitche Are you looking at the test failures for this PR?

**Comment by stephentoub** [comment]
I took a quick look at them in the CI report; I believe they're all related to #846 and should simply be marked as [ActiveIssue(846, PlatformID.AnyUnix)] to suppress their execution on Linux and OSX.  Most of them are do to a lack of support for normalization, and one has to do with a lack of proper encoding support.

**Comment by davidsh** [comment]
@stephentoub How did the tests pass before this PR?  I.e., when the System.Net.Http tests were first added, the PR build was good.

**Comment by stephentoub** [comment]
> How did the tests pass before this PR?

I believe the primary issue Matt is fixing here is that the tests weren't actually being run then, or at least weren't being run on Linux, because of the scripting expecting a specific directory structure that this library didn't match.

**Comment by mmitche** [comment]
Yes that's the case.  These may need to be disabled on Linux for now.

**Comment by davidsh** [comment]
ok.  Once you fix the tests (or disable them), then LGTM.

**Comment by mmitche** [comment]
@davidsh I'm having trouble getting the build to reference xunit.netcore.extensions (contains the ActiveIssue attribute) without floating the set of packages used with System.Net.Http.  The newest packages appear to require use of the newest dnx.exe, I've PR'd the update to DNX as #2052 and am waiting on that to test again.

**Comment by mmitche** [comment]
@davidsh Currently blocked on the dnx update

**Comment by davidsh** [comment]
@dotnet-bot test this please

**Comment by mmitche** [comment]
@davidsh I might have to do some tweaks, not exactly sure what state I left this branch.  Let me give it a quick try and we'll see how it goes.

**Comment by davidsh** [comment]
@mmitche The build finished successfully.  So, I think this can be merged now.

**Comment by mmitche** [comment]
(thank goodness :))  Yes.

**Pull request merged**